pineappl/src/evolution.rs

```diff
@@ -552,11 +556,11 @@ pub(crate) fn evolve_slice_with_two(
     let mut operators_b = Vec::new();
 
     for subgrids_ol in grid.subgrids().axis_iter(Axis(1)) {
-        let mut tables = vec![Array2::zeros((info.x0.len(), info.x0.len())); channels0.len()];
+        let mut tables = vec![Array2::zeros((info_a.x0.len(), info_a.x0.len())); channels0.len()];
```

Comment:
Shouldn't this be `info_b` in the second case?

Reply:
If think here using `info_a` or `info_b` is exactly the same. But just to be always certain, I can add a check that this is always the case, and otherwise panic.

Reply:
I believe it's possible that different hadronic initial states have different `x` grids, which then would be reflected in different `infos`. I fixed most of that in commit 4a8f7a492118aea8abf74650c9e0757f6027c895.